gnosis/safe-react
# [L2-UX] Change Polygon RPC

## What it solves
Resolves #2852

## How this PR fixes it
Polygon RPC changed to newest aggregator.

## How to test it
Delete Polygon from MM config and then try to connect again via the Safe. The option to automatically add the Polygon config to MM should open.
